components/ButtonGenerator/index.tsx

```diff
@@ -74,7 +76,9 @@ export const initialButtonState: ButtonState = {
   editable: false,
   widget: false,
   disablePaymentId: false,
-  opReturn: ''
+  opReturn: '',
+  contributionOffset: 0,
+  disableAltpayment: false
```

Comment:
Make sure to include all the new props here as well with default values, otherwise they wont reset with the reset button. All the strings should just be '' and auto-close should be true